gson/src/main/java/com/google/gson/ExclusionStrategy.java

```diff
@@ -93,12 +93,14 @@
 public interface ExclusionStrategy {
 
   /**
+   * Method use to decide if a field should be serialized or deserialized
```

Comment:
I would write:

```java
 * Decides if a field should be skipped during serialization or deserialization.
 *
```

The main changes:
* blank line before the `@` tags;
* no need to say that this is a method, just what it does;
* explicitly talk about "skipping" since it is sort of the opposite of what you might expect from a method that is "deciding".